### ISSUE TITLE
Licensee/linguist warning is noisy

### ISSUE THREAD

**Issue by hyandell** [todogroup/repolinter#265]
## Issue Report

If linguist/licensee are not installed, repolinter outputs a warning:

```
Axiom language failed to run with error: Linguist not installed
Axiom license failed to run with error: Licensee not installed
```

This is confusing for a user if the ruleset did not require these.

## Expected Behavior

When the ruleset does not have language/license specific rules, do not warn.

## Actual Behavior

Warning happens.

## Steps to Reproduce the Issue

Use  repolinter without linguist/licensee installed.
